xml/System.Runtime.CompilerServices/ExtensionMarkerAttribute.xml

```diff
@@ -28,7 +28,10 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>
+      <para>This attribute is used to mark extension members and associate them with a specific marker type (which provides detailed information about an extension block and its receiver parameter).</para>
+      <para>This attribute should not be used by developers in source code.</para>
```

Comment:
Should we use the more standard "This attribute is intended for internal use only." text?

Reply:
I had a look for precedent in https://github.com/search?q=repo%3Adotnet%2Fdotnet-api-docs%20path%3Axml%2FSystem.Runtime.CompilerServices%20internal&type=code

I see that one thing we do is add `<forInternalUseOnly />`

Which inserts
> This API supports the product infrastructure and is not intended to be used directly from your code.

```suggestion
      <forInternalUseOnly />
```